### PR TITLE
download-eeglab-plugin: Add script to install plugin w/o GUI

### PR DESCRIPTION
**Pull request by keggsmurph21**
While doing some data processing on an EC2 instance, I needed a way
to install an eeglab plugin.  Since I didn't have access to a GUI,
I realized that I could achieve this by using the "plugin_install"
function.

I wrote this script to wrap around that function (that also queries
the plugin listing website for version / download link), and I figured
someone else might find it useful!  :^)

### CODE REVIEW

**Comment by keggsmurph21** [comment]
Not sure if there's an official method for opening PRs, but please let me know if this isn't the correct location for this script!

File: functions/supportfiles/download-eeglab-plugin [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+# Use python to query the sccn.ucsd.edu plugin website for the <version> and
+# <link> for a given plugin <name> (if it exists).  If we find a match, we'll
+# write it to a temporary files and read it back afterwards.
+VERSION_FILE="/tmp/plugin-version-$(uuidgen)"
+LINK_FILE="/tmp/plugin-link-$(uuidgen)"
+(
+python3 - <<PYTHON_SCRIPT
+
+import bs4
+import requests
```
Comment (keggsmurph21) [inline functions/supportfiles/download-eeglab-plugin:33]: This depends on the python packages [bs4](https://pypi.org/project/beautifulsoup4/) and [requests](https://pypi.org/project/requests) being installed on the user's system.  In general, these can be installed via
```bash
$ python3 -m pip install --user bs4 requests
```

File: functions/supportfiles/download-eeglab-plugin [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        break
+    elif [[ "$response" =~ [nN](o)? ]]; then
+        echo "ok, quitting!"
+        exit
+    fi
+    echo "invalid response '$response', expected 'y' or 'n'"
+done
+
+# Do the import and then quit :^)
+matlab <<MATLAB_SCRIPT
```
Comment (keggsmurph21) [inline functions/supportfiles/download-eeglab-plugin:91]: This depends on `matlab` being on the user's `$PATH`.

File: functions/supportfiles/download-eeglab-plugin [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+usage: $0 <plugin-name>
+
+    Install a plugin called <plugin-name> from the sccn.ucsd.edu website.
+
+USAGE
+    exit 1
+fi
+
+# Go to the root of this repository so that matlab imports work correctly.
+cd "$(git rev-parse --show-toplevel)"
```
Comment (keggsmurph21) [inline functions/supportfiles/download-eeglab-plugin:19]: This depends on the installation containing the actual `git` repository (not sure if that's the case for the official installer, as I just `git clone`ed this onto my system).

**Comment by arnodelorme** [comment]
What about 

plugin_askinstall('bva-io', [], true);

This is how we install from the command line. Does that work on AWS?

**Comment by keggsmurph21** [comment]
😅😅 Looks like that works! Not sure how I didn't stumble across that myself. Thanks for the quick response though 🙂

**Comment by arnodelorme** [comment]
I will document this.

**Pull request closed without merging**